dell-csi-helm-installer/csi-install.sh

```diff
@@ -15,7 +15,7 @@ PROG="${0}"
 NODE_VERIFY=1
 VERIFY=1
 MODE="install"
-DEFAULT_DRIVER_VERSION="2.0.0"
+DEFAULT_DRIVER_VERSION="v2.0.0"
```

Comment:
Driver version to be changed to 2.3.0?

Reply:
Nope, this should be `v2.0.0` only after this new helm change, Same format is maintained across all the drivers

Reply:
ok